sailthru_content/services/sailthru_translation_service.py

```diff
@@ -0,0 +1,275 @@
+import datetime
+import logging
+
+
+class SailthruTranslationService(object):
+    def __init__(self, data_service, lms_root, fixups=None):
+        self.data_service = data_service
+        self.lms_root = lms_root
+        self.fixups = fixups
+
+    def _convert_date(self, iso_date):
+        """ Convert date from ISO 8601 (e.g. 2016-04-15T20:35:11.424818Z) to Sailthru format
+        """
+        if iso_date is None:
+            return None
+        try:
+            if '.' in iso_date:
+                return datetime.datetime.strptime(iso_date, "%Y-%m-%dT%H:%M:%S.%fZ").strftime("%Y-%m-%d")
+            return datetime.datetime.strptime(iso_date, "%Y-%m-%dT%H:%M:%SZ").strftime("%Y-%m-%d")
+        except ValueError:
+            return None
+
+    def _convert_datetime(self, iso_date):
```

Comment:
You can collapse this method into the one above. They share most of their code, with the parsing being the only difference.